rasa/cli/data.py

```diff
@@ -480,11 +495,37 @@ def _assert_two_stage_fallack_policy_is_migratable(config: Dict):
             f"'{USER_INTENT_OUT_OF_SCOPE}' before migrating the model configuration. "
         )
 
+    if (
+        two_stage_fallback_config.get(
+            "fallback_nlu_action_name", ACTION_DEFAULT_FALLBACK_NAME
+        )
+        != ACTION_DEFAULT_FALLBACK_NAME
+    ):
+        rasa.shared.utils.cli.print_error_and_exit(
+            f"The TwoStageFallback in Rasa Open Source 2.0 has to use the action "
```

Comment:
```suggestion
            f"The `TwoStageFallbackPolicy` in Rasa Open Source 2.0 has to use the action "
```

Reply:
it's not a policy any more

Reply:
what is it then? not the `TwoStageFallbackAction` right? if it's just a "two-stage fallback" it should be
```suggestion
            f"The two-stage fallback in Rasa Open Source 2.0 has to use the action "
```

Reply:
the name of what thing exactly? if it's just a proper noun in Rasa, it would still be
```suggestion
            f"The Two-Stage Fallback in Rasa Open Source 2.0 has to use the action "
```
writing it as one word makes it look like a class or an existing concept in Rasa - but it's just called "Two-Stage Fallback" elsewhere in the docs